app/models/user_auth_token.rb

```diff
@@ -220,6 +222,16 @@ def rotate!(info = nil)
     end
 
   end
+
+  def self.enforce_session_count_limit!(user_id)
+    tokens_to_destroy = where(user_id: user_id).
+      where('rotated_at > ?', SiteSetting.maximum_session_age.hours.ago).
+      order("rotated_at DESC").offset(MAX_SESSION_COUNT)
+
+    count = tokens_to_destroy.delete_all # Returns the number of deleted rows
+
+    count
```

Comment:
this `count` is redundant. In fact, no `count` variable is required because ruby will return the result from `delete_all` - if you're worried it's not clear the comment is there, plus you have a test!

Reply:
Ah yes, I added the count variable for some puts debugging and forgot about it. Will remove it